packages/batch/src/SqsFifoPartialProcessor.ts

```diff
@@ -94,16 +153,44 @@ class SqsFifoPartialProcessor extends BatchProcessorSync {
     const remainingRecords = this.records.slice(firstFailureIndex);
 
     for (const record of remainingRecords) {
-      const data = this.toBatchType(record, this.eventType);
-      processedRecords.push(
-        this.failureHandler(data, new SqsFifoShortCircuitError())
-      );
+      this.processFailRecord(record, new SqsFifoShortCircuitError());
     }
 
     this.clean();
 
     return processedRecords;
   }
+
+  /**
+   * Adds the specified group ID to the set of failed group IDs.
+   * @param group - The group ID to be added to the set of failed group IDs.
+   */
+  private addToFailedGroup(group: string): void {
```

Comment:
I think we can convert these to JS private fields, i.e. `private addToFailedGroup` -> `#addToFailedGroup`.

I would default to that and eventually open them up to `protected` if there is demand.

Reply:
To be clear, should I change it for all private methods/variables of this file? 

I can see a similar pattern is followed in `Parser.ts` from `jmsepath`
https://github.com/aws-powertools/powertools-lambda-typescript/blob/main/packages/jmespath/src/Parser.ts

@dreamorosi

Reply:
Only the ones that you're adding in this PR for now